### PR TITLE
flow : fixup reflection by using name prefix XXX_ (protobuf)

### PR DESCRIPTION
**Pull request by nplanel**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -36,7 +36,7 @@ all: install
 	# do not export LastRawPackets used internally
 	sed -e 's/json:"LastRawPackets,omitempty"/json:"-"/g' -i flow/flow.pb.go
 	# add flowState to flow generated struct
-	sed -e 's/type Flow struct {/type Flow struct {\n\tstate flowState/' -i flow/flow.pb.go
+	sed -e 's/type Flow struct {/type Flow struct {\n\tXXX_state flowState/' -i flow/flow.pb.go
```
Comment (safchain) [inline Makefile]: `json:"-"` should be added otherwise this field will be seen by user

Comment (nplanel) [reply Makefile]: done

**Comment by nplanel** [comment]
run skydive-unit-tests

**Comment by nplanel** [comment]
done

**Pull request merged**